docs/cli/usage.md

```diff
@@ -94,18 +94,30 @@ Or for just one calculation ...
 poetry run ths_ds_check rlzs ./WORKING/ARROW/DS1_DFG/ ./WORKING/ARROW/DS2_DFG/ -l2 -x -v -n 5 -cid T3BlbnF1YWtlSGF6YXJkU29sdXRpb246NjkzMTg5NA==
 ```
 
-### DEMO
+### DEMO (with both local and S3 filesystems)
 
-## prep the producer
+Note that local file paths `./WORKING/ARROW/DSR-DEMO` below may be replaced with a valid S3 bucket/prefix e.g `s3://ths-poc-arrow-test/DSR-DEMO`.
+
+In both cases that part of the path is the folder, which for targets, will be created if it doesn't exist.
```

Comment:
not sure what "that" refers to in the sentence; may need a re-write.

Reply:
yep, fixed that - thanks